File: Facultate/Anul 2/Sem 2/MPP/Laburi/Lab_2_All/Lab_3/src/main/java/models/Employee.java
package models;

public class Employee extends Entity<Long>{

    private final String username;
    private final String password;
    private final String name;
    private final String office;

    public Employee(String username, String password, String name, String office) {
        this.username = username;
        this.password = password;
        this.name = name;
        this.office = office;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }

    public String getName() {
        return name;
    }

    public String getOffice() {
        return office;
    }
}

File: Facultate/Anul 2/Sem 2/MPP/Laburi/Lab3/CSharpLab3/CSharpLab3/models/Employee.cs
namespace CSharpLab3.models;

public class Employee : Entity<long>
{
    public required string Username { get; init; }
    public required string Password { get; init; }
    public required string Name { get; init; }
    public required string Office { get; init; }

    public override string ToString()
    {
        return $"Employee: Username = {Username}, Name = {Name}, Office = {Office}";
    }
}